### PR TITLE
SSH terminal for targets

### PR DESCRIPTION
**Pull request by xisui-MSFT**
SSH terminal for targets:
sshterminal

### CODE REVIEW

File: Extension/src/Debugger/extension.ts [hunk 1]
```diff
@@ -130,6 +131,12 @@ export function dispose(): void {
     disposables.forEach(d => d.dispose());
 }
 
+function sshTerminal(node: TargetLeafNode): void {
+    const terminal: vscode.Terminal = vscode.window.createTerminal(`SSH: ${node.name}`);
+    terminal.sendText(`ssh ${node.name}`);
```
Comment (merlynomsft) [inline Extension/src/Debugger/extension.ts]: To consider: Can 'name' contain any characters that we'd need to quote or escape?

**Review by merlynomsft — APPROVED** [review]

File: Extension/src/Debugger/extension.ts (hunk 1 above)
Comment (xisui-MSFT) [reply Extension/src/Debugger/extension.ts]: Strings are escaped automatically.

Comment (xisui-MSFT) [reply Extension/src/Debugger/extension.ts]: Did some tests
- Quotes in names are not supported at all in SSH config, so nothing will show up in our view.
- Everything after spaces in names are omitted

So we only need to add quotes around node.name to make the corner cases work.

**Review by olegkhar — APPROVED** [review]

File: Extension/package.nls.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
     "c_cpp.command.selectSshTarget.title": "Select SSH target",
     "c_cpp.command.activeSshTarget.title": "Get the active SSH target",
     "c_cpp.command.refreshCppSshTargetsView.title": "Refresh",
+    "c_cpp.command.sshTerminal.title": "Connect to this target in a new terminal",
```
Comment (sean-mcmanus) [inline Extension/package.nls.json]: Should "target" be "SSH target"? I'm concerned the translators may not know how to translate it consistently with the other references to "SSH target" in the other strings. Maybe a "comment" to translators could be added instead.

File: Extension/package.json [hunk 3]
```diff
@@ -5073,12 +5079,17 @@
...
           "when": "viewItem == CppSshTargetsView.targetLeafRemovable || viewItem == CppSshTargetsView.targetLeafRemovableCanSetActive",
-          "group": "inline@1"
+          "group": "inline@2"
         },
         {
           "command": "C_Cpp.setActiveSshTarget",
           "when": "viewItem == CppSshTargetsView.targetLeafCanSetActive || viewItem == CppSshTargetsView.targetLeafRemovableCanSetActive",
           "group": "inline@0"
+        },
+        {
```
Comment (sean-mcmanus) [inline Extension/package.json:5084]: Would it be better to sort these by the inline@ number?

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

**Review by sean-mcmanus — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
@xisui-MSFT You want this checked in for 1.13.1?

**Comment by xisui-MSFT** [comment]
> @xisui-MSFT You want this checked in for 1.13.1?

Sure if that fits the release schedule.

**Pull request merged**